src/Ledger/PDF.lagda

```diff
@@ -10,6 +10,10 @@
 
 module Ledger.PDF where
 
+open import Data.Maybe.Properties.Ext
+-- ^ This module is not used but we import it here for now
+--   so that the CI doesn't complain about an unused module.
+
```

Comment:
Why does the CI fail when there are unused modules?

Reply:
i.e. the check perfomed by `src/checkTypechecked.sh` seems superfluous to me and leads to the unpleasant fix above.

Reply:
Yes, I don't like that feature of the script either.  In fact, I don't like that we rely on a script to make sure everything type-checks.  I raised this issue before and proposed a more standard alternative, but was voted down.

In my `agda-algebras` library, in Jacques' `agda-categories` library, and I believe also in the Agda standard library, the `Everything.agda` file is automatically generated and includes all modules in the library.  This makes it trivial to check *directly* whether everything type-checks, rather than relying on a script to do it in the CI after the changes are pushed to GitHub.

However, I believe Andre prefers to keep the `Everything.agda` file very minimal and clean, so really it has a different purpose than a typical "everything" file.

Our Everything.agda file is not an "everything" file and, imho, it should be renamed to something more intuitive, like `FormalLedger.agda`.

Perhaps @WhatisRT would willing to reconsider this at some point...?

Reply:
Actually, I retract the statement, "I don't like that we rely on a script to make sure everything type-checks."  I'm happy that the CI double checks that everything type-checks.  What I don't like is that it complains if there's a module that isn't yet imported anywhere... but, with the current set up, that's how we confirm that everything in the library type-checks (and is useful for something)---an error like the one above is the script's way of telling us that (a) we have a module that's not used and (b) such a module may not have been type-checked yet (since our `Everything.agda` does not depend on it).